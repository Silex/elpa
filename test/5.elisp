;;; Relint test file 5          -*- emacs-lisp -*-

;; Test let bindings
(defun test-let-inside (x y)
  (looking-at (let ((x "a")
                    (y "b"))
                (let* ((y "^")
                       (z (concat x y)))
                  z))))

(defun test-let-outside (x y)
  (let ((x "A")
        (y "B"))
    (let* ((y "^")
           (z (concat x y)))
      (looking-at z))))

;; Test setq
(defun test-setq-inside (x)
  (looking-at (setq x "[AA]")))

(defun test-setq-outside (x c)
  (setq x "[")
  (let ((y "B")
        (z "M"))
    (setq z "B")
    (looking-at (concat x y z "]"))))
