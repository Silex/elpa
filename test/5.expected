5.elisp:5:15: In call to looking-at: Unescaped literal `^' (pos 1)
  "a^"
   .^
5.elisp:16:19: In call to looking-at: Unescaped literal `^' (pos 1)
  "A^"
   .^
5.elisp:20:15: In call to looking-at: Duplicated `A' inside character alternative (pos 2)
  "[AA]"
   ..^
5.elisp:27:17: In call to looking-at: Duplicated `B' inside character alternative (pos 2)
  "[BB]"
   ..^
